utils.py

```diff
@@ -721,6 +721,40 @@ def get_hashable_value(value):
         return value
 
 
+def find_all_values_for_key(key, value_set):
```

Comment:
I think this approach isn't foolproof. See e.g. https://github.com/oppia/oppia/blob/develop/data/explorations/welcome.yaml. There are some places like multiple-choice input options (like "It's translated from a different language.") that should be caught here but I suspect aren't, because they don't correspond to a 'html' key.

So I would suggest a different approach. Implement exp_domain.Exploration.get_all_html_content_strings() that returns a list of all the HTML content in the exploration. It should in turn call Interaction.get_all_html_content_strings() etc. Write tests for that too.

@ishucr7's comment [here](https://github.com/oppia/oppia/pull/4965#issuecomment-391015812) might help you as well, but make sure to do your own independent verification to ensure that you've covered everything.

Reply:
Hi Sean, 
Sorry was delayed response on this (was implementing the migration for Ckeditor and Textangular). I found html content at:
- content in state
- default outcome in interactions
- solution in interaction
- hints in interaction
- feedback in outcome -> answer_group -> interaction
- rule_spec input in answer group (for ItemSelectionInput)
- customization_arg value in interaction (for ItemSelectionInput, MultipleChoiceInput)

Reply:
Done!